src/core_modules/capture-core/components/Pages/EnrollmentAddEvent/EnrollmentAddEventPage.component.js

```diff
@@ -40,30 +55,36 @@ const EnrollmentAddEventPagePain = ({
             <div className={classes.title}>
                 {i18n.t('Enrollment{{escape}} New Event', { escape: ':' })}
             </div>
-            <div>
-                <Widget
-                    header={
-                        <div className={classes.header}>
-                            {icon && (
-                                <div className={classes.icon}>
-                                    <NonBundledDhis2Icon
-                                        name={icon?.name}
-                                        color={icon?.color}
-                                        width={30}
-                                        height={30}
-                                        cornerRadius={2}
-                                    />
-                                </div>
-                            )}
-                            <span> {stageForm.name} </span>
+            <div className={classes.contentWrapper}>
+                <div className={classes.formWrapper}>
+                    <Widget
+                        header={
+                            <div className={classes.header}>
+                                {icon && (
+                                    <div className={classes.icon}>
+                                        <NonBundledDhis2Icon
+                                            name={icon?.name}
+                                            color={icon?.color}
+                                            width={30}
+                                            height={30}
+                                            cornerRadius={2}
+                                        />
+                                    </div>
+                                )}
+                                <span> {stageForm.name} </span>
+                            </div>
+                        }
+                        noncollapsible
+                    >
+                        <div data-test="edit-event-form">
+                            <EditEventDataEntry formFoundation={programStage.stageForm} />
```

Comment:
Hey, 
The `EditEventDataEntry` is the form component used in the edit page. For the new event page the old components are `SingleEventRegistrationEntry` and `NewEventDataEntry`. I think you should use these ones, not the ones from the edit form. The changes in the `.actions`, `.epics`, and rules engine helpers should also happen for the add event components, not the edit ones. WHYT?
Thanks!